src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -147,6 +352,15 @@ CHIP_ERROR ScenesServer::Init()
         {
             ChipLogDetail(Zcl, "ERR: setting LastConfiguredBy on Endpoint %hu Status: %x", endpoint, status);
         }
+
+        // Get Scene Table for a specific endpoint
+        sceneTable = scenes::GetSceneTableImpl(endpoint);
+        // Initialize the FabricSceneInfo list from data in Flash
```

Comment:
At this point, the `SceneInfoStructs` will get their `SceneCount` and `RemainingCapacity` from the SceneTable data stored in flash. I will explicitly state that since the comment is not clear.